### PR TITLE
connect: add language flag

### PR DESCRIPTION
**Pull request by oleg-jukovec**
The `-l` or `--language` flag allows a user to switch default
language for a connection. A value can be `lua` or `sql`.

Part of https://github.com/tarantool/tt/issues/58

The patchset also replaces Tarantool 2.8 to 2.10 on CI as a last release version.

### CODE REVIEW

File: cli/connect/console.go [hunk 1]
```diff
@@ -242,6 +271,16 @@ func getCompleter(console *Console) prompt.Completer {
 			return nil
 		}
 
+		switch console.language {
+		case DefaultLanguage:
```
Comment (psergee) [inline cli/connect/console.go:275]: Let's merge them:
`case DefaultLanguage, LuaLanguage:`

File: cli/connect/language.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+type Language int
+
+const (
+	DefaultLanguage Language = iota
+	LuaLanguage
+	SQLLanguage
+)
+
+func GetLanguage(str string) (Language, bool) {
```
Comment (psergee) [inline cli/connect/language.go:26]: Add comment.

File: cli/connect/language.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+		return DefaultLanguage, true
+	case luaStr:
+		return LuaLanguage, true
+	case sqlStr:
+		return SQLLanguage, true
+	}
+	return DefaultLanguage, false
+}
+
+func (l Language) String() string {
```
Comment (psergee) [inline cli/connect/language.go:39]: Add comment.

**Review by psergee — APPROVED** [review]
Thanks for the patch! Just minor notes.

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        tarantool-version: ["1.10", "2.8"]
+        tarantool-version: ["1.10", "2.8", "2.10"]
```
Comment (LeonidVas) [inline .github/workflows/tests.yml:20]: I think it is enough to run the tests on LTS and the latest version of tarantool.

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -97,7 +97,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        sdk-version: ["2.8.2-0-gfc96d10f5-r421"]
+        sdk-version: ["2.8.2-0-gfc96d10f5-r421", "2.10.2-0-gf4228cb7d-r508-linux-x86_64"]
```
Comment (LeonidVas) [inline .github/workflows/tests.yml:100]: I propose to use only one version of the sdk for the tests, the latest one.

File: cli/connect/console.go [hunk 6]
```diff
@@ -177,11 +185,32 @@ func loadHistory(console *Console) error {
...
-			return
+		switch console.language {
+		case DefaultLanguage:
+			fallthrough
+		case LuaLanguage:
+			if !inputIsCompleted(console.input, console.luaState) {
+				console.livePrefixEnabled = true
+				return
+			}
+		case SQLLanguage:
```
[the remark this answers is not in the document]
Comment (oleg-jukovec) [reply cli/connect/console.go]: Replaced by if.

File: cli/connect/console.go [hunk 7]
```diff
@@ -242,6 +271,16 @@ func getCompleter(console *Console) prompt.Completer {
 			return nil
 		}
 
+		switch console.language {
+		case DefaultLanguage:
+			fallthrough
+		case LuaLanguage:
+		case SQLLanguage:
+			// Tarantool does not implements autocompletion for SQL:
```
[the remark this answers is not in the document]
Comment (oleg-jukovec) [reply cli/connect/console.go]: Thank you, fixed!

File: cli/connect/console.go [hunk 8]
```diff
@@ -177,11 +185,32 @@ func loadHistory(console *Console) error {
...
+			return
+		}
+
 		console.input += in + " "
 
-		if !inputIsCompleted(console.input, console.luaState) {
-			console.livePrefixEnabled = true
-			return
+		switch console.language {
+		case DefaultLanguage:
```
[the remark this answers is not in the document]
Comment (oleg-jukovec) [reply cli/connect/console.go]: Replaced by if.[2]

File: cli/connect/console.go (hunk 1 above)
Comment (oleg-jukovec) [reply cli/connect/console.go]: Replaced by if.[3]

File: cli/connect/language.go (hunk 2 above)
Comment (oleg-jukovec) [reply cli/connect/language.go]: Thanks, done.

File: cli/connect/language.go (hunk 3 above)
Comment (oleg-jukovec) [reply cli/connect/language.go:43]: Thank you, done.

File: .github/workflows/tests.yml (hunk 4 above)
Comment (oleg-jukovec) [reply .github/workflows/tests.yml]: I've replaced it by `["1.10", "2.10"]`.

File: .github/workflows/tests.yml (hunk 5 above)
Comment (oleg-jukovec) [reply .github/workflows/tests.yml]: I left only `["2.10.2-0-gf4228cb7d-r508-linux-x86_64"]`

**Review by LeonidVas — APPROVED** [review]

**Pull request merged**